app/MMB/src/scenes/tripServices/insurance/MoreInfoScene.js

```diff
@@ -0,0 +1,374 @@
+      />,
+      <Translation key="missed_flight_coverage-1-—" passThrough="—" />,
+      <Translation key="missed_flight_coverage-2-100 €" passThrough="100 €" />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.flight_cancellation"
+        id="mmb.trip_services.insurance.selection.more_info.flight_cancellation"
+      />,
+      <Translation
+        key="flight_cancellation-1-booking.passenger.insurance.to_amount_paid_for_flights_and"
+        id="mmb.trip_services.insurance.selection.more_info.to_amount_paid_for_flights_and"
+      />,
+      <Translation
+        key="flight_cancellation-2-booking.passenger.insurance.to_amount_paid_for_flights_and"
+        id="mmb.trip_services.insurance.selection.more_info.to_amount_paid_for_flights_and"
+      />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.co_insurance_causes_flights_cancellation"
+        id="mmb.trip_services.insurance.selection.more_info.co_insurance_causes_flights_cancellation"
+      />,
+      <Translation
+        key="co_insurance_causes_flights_cancellation-1-10%"
+        passThrough="10%"
+      />,
+      <Translation
+        key="co_insurance_causes_flights_cancellation-2-10%"
+        passThrough="10%"
+      />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.co_insurance_causes_subsequent_services"
+        id="mmb.trip_services.insurance.selection.more_info.co_insurance_causes_subsequent_services"
+      />,
+      <Translation
+        key="co_insurance_causes_subsequent_services-1-20%"
+        passThrough="20%"
+      />,
+      <Translation
+        key="co_insurance_causes_subsequent_services-2-20%"
+        passThrough="20%"
+      />,
+    ],
+  ];
+
+  function onPress() {
+    props.navigation.navigate({
+      routeName: 'mmb.trip_services.insurance.selection.more_info.terms',
+      key: 'key-mmb.trip_services.insurance.selection.more_info.terms',
+    });
+  }
+
+  return (
+    <ScrollView style={styles.scrollView}>
+      <Text style={[styles.smallerFont, styles.headerColor, styles.pitch]}>
+        <Translation id="mmb.trip_services.insurance.selection.more_info.pitch" />
+      </Text>
+      {rows.map((row, index) => (
+        <Row key={`row-${index + 1}`} index={index} data={row} />
+      ))}
+      <Text style={[styles.smallerFont, styles.headerColor, styles.pitch]}>
+        <Translation id="mmb.trip_services.insurance.selection.more_info.agree_with_terms" />
+      </Text>
+      <View style={styles.terms}>
+        <LinkButton
+          title={
+            <Translation id="mmb.trip_services.insurance.selection.more_info.terms" />
+          }
+          titleStyle={styles.smallerFont}
+          onPress={onPress}
+        />
+      </View>
+    </ScrollView>
+  );
+};
+
+export default MoreInfoScene;
+
+type RowProps = {|
+  +data: [
+    React.Element<typeof Translation>,
```

Comment:
What are these props in the array? Could you rather pass them as an object with some naming?